### PR TITLE
download_shard: validate field deprecated

### PR DESCRIPTION
**Pull request by mDuo13**
Documents https://github.com/ripple/rippled/pull/3252

Adds commandline syntax examples for the method.

### CODE REVIEW

File: content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md [hunk 1]
```diff
@@ -45,15 +45,23 @@ An example of the request format:
...
 
-The request includes the following fields:
+The request includes the following field:
 
 | `Field`    | Type    | Description                                           |
 |:-----------|:--------|:------------------------------------------------------|
 | `shards`   | Array   | List of Shard Descriptor objects (see below) describing shards to download and where to download them from. |
-| `validate` | Boolean | _(Optional)_ If `false`, skip validating the downloaded data. The default is `true`, which checks that the shard in the archive contains all the data objects for the shard and the shard is part of the ledger history of the current validated ledger. |
+
+The `validate` field is deprecated and may be removed in a future version. (The server always checks the integrity of shards when it imports them.) [Updated in: rippled 1.6.0][]
```
Comment (undertome) [inline content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md:64]: Not super familiar with markdown, why the empty square brackets at the end?

File: content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # download_shard
 [[Source]](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/DownloadShard.cpp "Source")
 
-Instructs the server to download a specific [shard of historical ledger data](history-sharding.html) from an external source. Your `rippled` server must be [configured to store history shards](configure-history-sharding.html). [New in: rippled 1.1.0][]
+Instructs the server to download a specific [shard of historical ledger data](history-sharding.html) from an external source. Your `rippled` server must be [configured to store history shards](configure-history-sharding.html). [Updated in: rippled 1.6.0][]
```
Comment (undertome) [inline content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md:4]: More empty brackets - is this intentional?

**Review by seelabs — APPROVED** [review]
:+1: LGTM.

**Comment by undertome** [comment]
I noticed this line at the bottom of the md file:
`{% include '_snippets/rippled-api-links.md' %} {% include '_snippets/tx-type-links.md' %} {% include '_snippets/rippled_versions.md' %}`

Is this supposed to parsed like this?

File: content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md (hunk 1 above)
Comment (mDuo13) [reply content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md:64]: It's a link that's defined elsewhere. (The [actual file with the definitions](https://raw.githubusercontent.com/ripple/xrpl-dev-portal/v1.6.0/content/_snippets/rippled_versions.md) is imported into this one using some non-standard syntax.)

File: content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md (hunk 2 above)
Comment (mDuo13) [reply content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md:4]: Yes, see above.

**Comment by mDuo13** [comment]
The `{% include %}` syntax is something our doc tool ([Dactyl](https://github.com/ripple/dactyl/)) parses when building the site, so short answer, yes, that's to be expected.

**Pull request merged**